content/docs/packit-service.md

```diff
@@ -29,6 +29,21 @@ in all current Fedora OS releases via [Testing Farm](/docs/testing-farm).
 Packit is also available as a [CLI tool](https://github.com/packit/packit/blob/main/README.md),
 so you can always try things locally on your own. Note that testing is not currently supported with the CLI tool.
 
+
+## Requirements for running Packit Service jobs
+
+As a first step, you need to have a valid [Fedora Account System](https://fedoraproject.org/wiki/Account_System) account to be approved by us and start using Packit Service.
+Besides that:
+* If you want to configure builds via Packit Service, your software needs to comply with [Copr guidelines](https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr) 
+ since we use Fedora Copr for the builds. Therefore, please, make sure you read them before configuring the [Packit Service build job](/docs/configuration/#copr_build).
+* If you are interested in using [internal instance of the Testing Farm](https://docs.testing-farm.io/general/0.1/services.html#_red_hat_ranch)
+to run your tests in, please, [reach out to us](/#contact), since for this job, an additional approval on our side is needed.
+* For retrying the build and test jobs via `/packit build` and `/packit test` pull request comments, you need to have write access to the repository or be the 
```

Comment:
I'm not sure whether  `write access` or `commit access` is more clear/correct, WDYT?

Reply:
`write access` +1